### DISCUSSION TITLE
Why is LC12S used or needed?

### DISCUSSION THREAD

**Discussion by FrankVloemans** [Yogui79/IntexPureSpa#43] [Q&A] [answered]
Hi,
I try to understand your solution. LC12S seems to be needed for using bluetooth to communicate with the Intex Spa.
Why can't the bleutooth functionality of esp32 be used?

Thanks,
Frank

PS: great project. Thanks for sharing it.

**Comment by LEECHER1** [comment, marked as answer]
Hey,

The LC12S is only a whirless Chip. This chip is in the Model 2020 pump and Remote Control installed. Is no Bluetooth.

We use the LC12s also to communicate with the Pumpe and Remote Control.

**Reply by FrankVloemans** [reply]
Thanks for the swift reply.
This means that this hardware is really needed. It can't be replaced by another chip? (LC12S is very hard to find)

**Reply by LEECHER1** [reply]
Sorry, this works only with this chip.

I know is it not so easy to buy it. Only made it in China.

**Reply by Yogui79** [reply]
@FrankVloemans  you can find it on aliexpress for a few euro but delivery is long
